index.html

```diff
@@ -9,9 +9,18 @@
 </head>
 <body>
     <div id="container">
-        <div id="image-background"></div>
+        <div id="image-background">
+            <div id="platforms">
+                <div id="platform"></div>
```

Comment:
Quando usamos id, o ideal é que só tenha um único elemento com esse id.
Se for usar vários, melhor trabalhar com class